File: junit/se/fredsberg/game/random/FixedScrambler.java
package se.fredsberg.game.random;

import java.util.List;

public class FixedScrambler implements Scrambler {

    private final int value;

    public FixedScrambler(int value) {
        this.value = value;
    }
    
    @Override
    public int random(int upperExclusiveLimit) {
        return value % upperExclusiveLimit;
    }

    @Override
    public void scramble(List<?> values) {
    }

}

File: jsunit/FixedScrambler.js
function FixedScrambler(value) {
	this.value = value;
};

FixedScrambler.prototype.random = function(upperExclusiveLimit) {
	return this.value % upperExclusiveLimit;
};

FixedScrambler.prototype.scramble = function(list) {
};
